src/main/webapp/js/navigation-loader.js

```diff
@@ -32,7 +32,7 @@ function addLoginOrLogoutLinkToNavigation() {
       .then((loginStatus) => {
         if (loginStatus.isLoggedIn) {
           navigationElement.appendChild(createLink(
-              '/user-page.html?user=' + loginStatus.username, 'Your Page'));
+              '/temp.html?user=' + loginStatus.username, 'feed'));
```

Comment:
If this hasn't been implemented yet, you can just remove it. No need to keep a link around that doesn't work